build-system/global-configs/experiments-config.json

```diff
@@ -6,6 +6,12 @@
     "expiration_date_utc": "2021-06-30",
     "define_experiment_constant": "NO_SIGNING_RTV"
   },
-  "experimentB": {},
+  "experimentB": {
+    "name": "amp-img in the deferred mount mode",
+    "environment": "AMP",
+    "issue": "https://github.com/ampproject/amphtml/issues/31915",
+    "expiration_date_utc": "2021-06-30",
+    "define_experiment_constant": "V1_IMG_DEFERRED_BUILD"
```

Comment:
Why is this already defined in `build-system/global-configs/experiments-const.json`?

Reply:
That's the new rule - the `experiment-const.json` now has to define the constant always and the `experiment-config.json` overrides it at built time for different scenarios.